### DISCUSSION TITLE
bump-formula-pr on Homebrew and Linuxbrew relationship

### DISCUSSION THREAD

**Discussion by sestegra** [Homebrew/discussions#1575] [Tap maintenance and Homebrew development]
Hello,

I just did a bump-formula-pr for kind 0.11.1 on homebrew-core.
Should I do the same on linuxbrew-core, or an automatic sync will run?

Kind regards,
Stéphane

**Comment by nandahkrishna** [comment]
Yes, I believe the `linuxbrew-core` maintainers will merge changes from homebrew/core (once a day if I'm not wrong). Note that this may be delayed due to an ongoing merge of the two repositories.
